SafetyFirst.Specs/ResultSpec.fs

```diff
@@ -249,6 +249,46 @@ let ``can use computation expressions to bind and map Results`` () =
       resultC = (Ok 15)
     @>
 
+[<Test>]
+let ``can use the applicative CE to combine errors together`` () = 
+  let tryToGetA = Ok 5
+  let tryToGetB = Ok 10
+  let add x y = x + y
+  let add' x y = Ok (x + y)
+
+  let resultA = 
+    validate (+) {
+      let! x = tryToGetA
+      and! y = tryToGetB
```

Comment:
I've never seen `and!` before

Reply:
Yeah this is a new F# 5 feature (https://docs.microsoft.com/en-us/dotnet/fsharp/whats-new/fsharp-50#applicative-computation-expressions)
It's akin to an `ado` in PureScript if you've ever used one of those.